nodejs-jquery/nodejs-jquery.spec

```diff
@@ -0,0 +1,57 @@
+%{?scl:%scl_package nodejs-%{npm_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global npm_name jquery
+
+Name: %{?scl_prefix}nodejs-%{npm_name}
+Version: 1.11.3
+Release: 1%{?dist}
+Summary: JavaScript library for DOM operations
+License: MIT
+Group: Development/Libraries
+URL: https://github.com/jquery/jquery.git
+Source0: http://registry.npmjs.org/jquery/-/jquery-1.11.3.tgz
+BuildRoot: %{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
+Requires: nodejs(engine)
```

Comment:
Sorry that last bit confused me:

As I understood your comment, there's no hard requirement on requiring the SCL nodejs engine/devel/packaging  instead of the non-SCL ones, but if I don't the files won't be in the SCL root. These packages are not in our SCL, so the only way I have to package it is on the 'global space' as nodejs-jquery. 

Maybe there's a way of putting %files in the SCL root manually on %install? My understanding is that if I package this as `nodejs-jquery` and Fedora/CentOS/RHEL decide to package it for the distro, the package from the distro would prevail and wipe this one out if it's newer.